Community/AssetManagement/lumeta_workflow_page.py

```diff
@@ -0,0 +1,311 @@
+# Copyright 2019 BlueCat Networks. All rights reserved.
+
+import ipaddress
+from flask import request, g, abort, jsonify
+from bluecat.api_exception import PortalException, APIException
+from bluecat import route, util
+from main_app import app
+
+# application config
+
+# Define global variable to hold handle to API object
+api = None
+
+#
+# GET, PUT or POST
+#
+@route(app, '/lumeta/getnetworklist', methods=['GET', 'PUT', 'POST'])
+@util.rest_workflow_permission_required('lumeta_workflow_page')
+@util.rest_exception_catcher
+def get_networks_get_networks_page():
+    # are we authenticated?
+    g.user.logger.info('SUCCESS')
+    configurations = None
+    configurations_json = []
+    if g.user:
```

Comment:
You don't need to check if g.user as you'd have been booted by the rest_workflow_permission_required decorator before getting into the function